tobiasHe/assignments/slidingwindow/lc340/lc340.js

```diff
@@ -0,0 +1,90 @@
+// Problem Statement #
```

Comment:
Nice job with comments but I think you could expand on your high level overview a little more. And don't forget to try typing those solutions twice to work on your muscle memory!